clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -130,7 +130,7 @@
     private MemberState state = MemberState.UNJOINED;
     private HeartbeatThread heartbeatThread = null;
     private RequestFuture<ByteBuffer> joinFuture = null;
-    private Generation generation = Generation.NO_GENERATION;
+    Generation generation = Generation.NO_GENERATION;
```

Comment:
If you agree with my other comments, this can be convert back to private.

Reply:
Fixed